torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -111,6 +112,26 @@ static at::Tensor to_dispatch(
 
 RegisterOperators reg(
     {Operator(
+         "prim::profile(Tensor t) -> ()",
+         [](const Node* node) {
```

Comment:
Capture precisely what is need, not the entire node. Technically we do not require the Graph to stay alive after creating code. Keep in mind this is the interpreter's internal graph, it has not responsibility to keep it alive. In this case both the ProfilingRecord and the Value* can be determined here and then captured by the lambda.

Reply:
fixed